lstchain/scripts/lstchain_r0g_to_r0v.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+"""
+Script to apply data volume reduction to R0G files in CTAR1 format (aka R1v1).
+Reduction is only applied to shower events (not to interleaved pedestal
+and flatfield).
+
+In principle it can work on R0 data (2 gains), but no gain selection will be 
+applied, and it does not make much sense to keep both gains on reduced 
+(pixel-selected) data!
+
+"""
+import logging
+import protozfits
+import argparse
+import sys
+import re
+
+from pathlib import Path
+from ctapipe.containers import EventType
+from ctapipe.io import read_table
+from ctapipe_io_lst import PixelStatus
+from astropy.io import fits
+
+import lstchain.paths as paths
+import numpy as np
+from contextlib import ExitStack
+
+parser = argparse.ArgumentParser(description="Volume reducer program (R0G to "
+                                             "R0V)")
+parser.add_argument('-f', '--R0G-file', dest='input_file', required=True,
+                    type=str, help='Input R0G file name (of stream 1)')
```

Comment:
you can directly use `type=Path` here to get the right type, also for the others

Reply:
Why not use positional arguments for the required ones?

Reply:
No obvious advantage, right? And it is similar in other scripts, I think (e.g. -f  for mandatory input files)